src/main/java/com/commercetools/project/sync/inventoryentry/InventoryEntrySyncer.java

```diff
@@ -67,7 +87,7 @@ public static InventoryEntrySyncer of(
   @Override
   protected CompletionStage<List<InventoryEntryDraft>> transform(
       @Nonnull final List<InventoryEntry> page) {
-    return CompletableFuture.completedFuture(replaceInventoriesReferenceIdsWithKeys(page));
+    return CompletableFuture.completedFuture(mapToInventoryEntryDrafts(page));
   }
 
   @Nonnull
```

Comment:
for the getQuery, I added a utility into InventoryReferenceResolutionUtils.

Reply:
Thanks, I didn't know about this new feature :) https://github.com/commercetools/commercetools-project-sync/pull/158/commits/975155d3130eb8a4cf7a9eace76468c2428ec5e8